astropy/stats/funcs.py

```diff
@@ -638,6 +689,17 @@ def poisson_conf_interval(n, interval='root-n', sigma=1):
                 conf_interval[0] = 0
         else:
             conf_interval[0,n==0] = 0
+    elif interval == 'kraft-burrows-nousek':
+        if conflevel is None:
+            raise ValueError('Set conflevel for method {0}. (sigma is ignored.)'.format(interval))
+        conflevel = np.asanyarray(conflevel)
+        if np.any(conflevel <=0) or np.any(conflevel >= 1):
+            raise ValueError('Conflevel must be a number between 0 and 1.')
+        background = np.asanyarray(background)
+        if np.any(background < 0):
+            raise ValueError('Background must be >= 1.')
```

Comment:
Sorry if I am mistaken, but, after reading the paper, I couldn't find a reason to explain why ``Background must be >= 1.``. Also, the code is checking for ``np.any(background < 0)``.